packages/strapi-design-system/src/v2/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Flex } from '../../Flex';
+import { VisuallyHidden } from '../../VisuallyHidden';
+import { Crumb } from './Crumb';
+import { CrumbLink } from './CrumbLink';
+
+export const Breadcrumbs = ({ label, children, ...props }) => {
+  return (
+    <Flex {...props}>
+      <VisuallyHidden>{label}</VisuallyHidden>
```

Comment:
What do you think about making this an `aria-label` or the `Flex` container? This would allow us to write:

```
<nav aria-label="Folder Structure Navigation" />
```

is is a very nice pattern for screenreader users to discover the structure of a page.

Reply:
I think it wouldn't be an issue - `div`s can reiceive an `arial-label` too. It's just less semantical.